docs/uns-usethenetwork/cli.md

```diff
@@ -305,4 +305,57 @@ CONTEXT:
 	readDateTime: 2019-08-27T08:45:40.000Z
 	height: 460
 
+```
+
+### `get-properties`
+
+#### Introduction
+Get properties of UNIK token.
+
+#### Parameters
+
+- `--unikid` (required): The UNIK token on which to set the properties.
+- `--confirmed` (optional): Minimum number of confirmation since the last update of the UNIK required to return the value. Default value is 3
```

Comment:
Can we add this notice?
"The command will fail if the minimum number of confirmations has not been reached yet."